Docker/README.md

```diff
@@ -27,8 +27,7 @@ docker run --gpus all -v /home/user/my_mri_data:/data \
                       --rm --user $(id -u):$(id -g) deepmi/fastsurfer:latest \
                       --fs_license /fs_license/license.txt \
                       --t1 /data/subjectX/t1-weighted.nii.gz \
-                      --sid subjectX --sd /output \
-                      --parallel
+                      --sid subjectX --sd /output
```

Comment:
Don't we have to put threads to 2 in order to have parallel behaviour?

Reply:
Definitely, I mean, we can also put 2 threads as the default ....

I just removed the --parallel flags because those are not really wanted/needed any more.